medmodels/medrecord/types.py

```diff
@@ -128,13 +128,15 @@ class AttributeInfo(TypedDict):
 class TemporalAttributeInfo(TypedDict):
     """Dictionary for a temporal attribute and its metrics."""
 
+    type: str
```

Comment:
should either be of type `AttributeType` or if it can only be of type `Temporal`, it should be `Literal["Temporal"]`